packages/story-editor/src/app/story/effects/useHistoryEntry.js

```diff
@@ -41,6 +42,13 @@ const ELEMENT_PROPS_TO_IGNORE = [
   'resource.font.variants',
   'resource.font.fallbacks',
   'resource.font.styles',
+  'resource.isOptimized',
+  'resource.length',
```

Comment:
Ignored for trim action .... from initial testing I think we still get 1 entry here (which we need) when the src changes.